sdk/src/claim.rs

```diff
@@ -288,6 +307,38 @@ impl Claim {
         }
     }
 
+    /// Create a new claim with a user supplied GUID.
+    /// user_guid: is user supplied guid conforming the C2PA spec for manifest names
+    /// claim_generator: User agent see c2pa spec for format
+    // #[cfg(not(target_arch = "wasm32"))]
```

Comment:
Why is this commented out? Can we remove before merging the PR?

Reply:
Does not seem to apply so I will remove.